src/components/AddToList.tsx

```diff
@@ -32,7 +33,7 @@ const AddToList: React.FC<IProps> = ({setPeople, people}) => {
                 age: parseInt(input.age),
                 img: input.img,
                 note: input.note
-            }
+            } as IPerson
```

Comment:
We should avoid type assertions if we can. It basically tells typescript "I know more than you do about this type", with the risk of being wrong 😄 .

We shouldn't need to tell typescript about the type of this object, as the `setPeople` handler only accepts `People` to be set.